src/bundle/ui-dev/src/modules/universal-discovery/components/bookmarks-list/bookmarks.list.js

```diff
@@ -37,6 +40,20 @@ const BookmarksList = ({ setBookmarkedLocationMarked, itemsPerPage }) => {
         [markedLocationId, loadedLocationsMap],
     );
     const [data, isLoading, reloadBookmarks] = useLoadBookmarksFetch(itemsPerPage, offset);
+    const containerClassName = createCssClassNames({
+        'c-bookmarks-list': true,
+        'c-bookmarks-list--no-items': bookmarks.length === 0,
```

Comment:
you could put `bookmarks.length` to extra variable, you're already using it in two places, but it's nitpick